### PR TITLE
Adds some pronoun helpers

### PR DESCRIPTION
**Pull request by ChangelingRain**
Applies said helpers to human and carbon examine, a few cult and clockcult files, and ignores everything else.

### CODE REVIEW

**Comment by ma44** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO (joan confirmed feminist)

**Comment by Incoming5643** [comment]
It's not a meme pull but the urge to meme is strong

**Review by Razharas — APPROVED** [review]
> Only three pronouns
> You arent allowed to choose yours at round start

Fucking white CIS queer-phobic racis pr

**Comment by TechnoAlchemisto** [comment]
admittedly people who play androgynous characters should be able to select they as a pronoun

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -206,14 +206,15 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
...
+		if(!is_servant_of_ratvar(C) && !C.null_rod_check() && number_legs) //you have legs right
 			C.apply_damage(noncultist_damage * 0.5, BURN, "l_leg")
 			C.apply_damage(noncultist_damage * 0.5, BURN, "r_leg")
 			if(C.m_intent != "walk")
 				if(!iscultist(C))
-					C << "<span class='warning'>Your legs shiver with pain!</span>"
+					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] shiver with pain!</span>"
 				else //Cultists take extra burn damage
-					C << "<span class='warning'>Your legs burn with pain!</span>"
+					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] burn with pain!</span>"
```
Comment (Cheridan) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: your leg burn with pain

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: super fixed

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -212,9 +212,9 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 			C.apply_damage(noncultist_damage * 0.5, BURN, "r_leg")
 			if(C.m_intent != "walk")
 				if(!iscultist(C))
-					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] shiver with pain!</span>"
+					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] shiver[number_legs < 2 ? "s":""] with pain!</span>"
 				else //Cultists take extra burn damage
-					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] burn with pain!</span>"
+					C << "<span class='warning'>Your leg[number_legs > 1 ? "s":""] burn[number_legs < 2 ? "s":""] with pain!</span>"
```
Comment (lordpidey) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: [number_legs > 1 ? "s burn" : " burns"]

**Pull request merged**